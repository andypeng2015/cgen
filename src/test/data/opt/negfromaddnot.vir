module foo

export function w $foo(w %x) {
@start:
  %x = not.w %x
  %x = add.w %x, 1_w
  ret %x
}
