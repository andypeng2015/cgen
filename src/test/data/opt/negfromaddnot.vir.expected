module foo

export function w $foo(w %x) {
@0:
  %0 = neg.w %x ; @3
  ret %0
}
